portal-api/src/constants/facilities.js

```diff
@@ -36,22 +36,10 @@ const DEAL_STATUS = {
   SUBMITTED_TO_UKEF: 'Submitted',
 };
 
-const TYPE = {
-  ADVANCE_PAYMENT_GUARANTEE: 'Advance payment guarantee',
-  BID_BOND: 'Bid bond',
-  MAINTENANCE_BOND: 'Maintenance bond',
-  PERFORMANCE_BOND: 'Performance bond',
-  PROGRESS_PAYMENT_BOND: 'Progress payment bond',
-  RETENTION_BOND: 'Retention bond',
-  STANDBY_LETTER_OF_CREDIT: 'Standby letter of credit',
-  WARRANTY_LETTER: 'Warranty letter',
-};
-
-module.exports = {
+export default {
```

Comment:
I dont think we really use `export` in `portal-api` javascript files. Happy for the change to go in because I much prefer it but flagging that its a bit inconsistent.
(Looks like outside of e2e-tests, we use `export default` 8 times in javascript files (all in UI scripts) vs the 1000+ `module.exports` )

Reply:
Though in future would it be possible for these kind of changes to come under a different ticket? Would help with reviewing & getting this in quicker :)

Reply:
Great shout @MarRobSoftwire , completely agree. I have reverted the portal-api changes / replaced instances of `export default` with `module.exports` to be consistent.